index.html

```diff
@@ -15,6 +15,7 @@ <h1 class="kermit">Index of Class Notes</h1>
 
     <h2>Daily Notes</h2>
     <ul>
+        <li><a href="notes/2018-09-25.html">September 24, 2018</a></li>
```

Comment:
Change date inside the <a> tag to September 25, 2018

Reply:
I think I fixed it.  Thank you!